gapis/perfetto/android/trace.go

```diff
@@ -97,6 +97,10 @@ func (p *Process) Capture(ctx context.Context, start task.Signal, stop task.Sign
 		return 0, err
 	}
 
+	if err := p.device.DeleteFile(ctx, perfettoTraceFile); err != nil {
```

Comment:
my understanding is this is not needed, gapis will end up calling `StartPerfettoTrace`

Reply:
I don't quite follow.  The purpose here is to delete the the using the same permissions as the creator of the trace to avoid any conflicts should the shell be un-rooted.

Reply:
Discussed offline :D